src/utils/createNewBoard.js

```diff
@@ -2,14 +2,23 @@
  * Boardを新規作成する
  * @param {number} x - boardの幅
  * @param {number} y - boardの高さ
+ * @param {string} gameMode - 対戦形式
  * @return {string[][]} - 新規作成したboard
  */
+import decideOrder from "./decideOrder";
 
-function createNewBoard(x, y) {
+function createNewBoard(x, y, gameMode) {
   let board = new Array(x);
   for (let i = 0; i < x; i++) {
     board[i] = new Array(y).fill(null);
   }
+  // cpu対戦の時、プレイヤーが後攻だったら、ボードの初期状態をcpuが一石置いた状態にしておく。
+  if (gameMode == "cpu") {
+    if (decideOrder() == false) {
```

Comment:
```suggestion
  if (gameMode === "cpu") {
    if (decideOrder() === false) {
```

ご存知かもしれませんが、JavaScriptでは等価演算子として、`===`と`==`があります。
厳密等価演算子（===）は型まで比較して同じかどうかチェックするので、基本的に同じ値かどうかチェックしたいときはイコール3つのほうを使うようにしましょう！！
参考：https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Strict_equality

余談ですが、===の使用を強制するESLintのルールもありますｗ

Reply:
すみません！意識して使えてなかったです、、
以後気をつけて厳密等価演算子使っていきます！